### ISSUE TITLE
32° F = 5.6843419E-14° C

### ISSUE THREAD

**Issue by mystyc** [doug-101/ConvertAll#4]
It is just as stated in the title.  
In _ConvertAll v0.8.0_ (windows 10), converting **_32° F_** into Celsius gives the strange result of **_5.6843419E-14° C_**.  This works in either order (1st box to 2nd or from 2nd to 1st).  Nonetheless, **_0° C_** still yields **_32° F_**.  This seems like a floating point error but it is so oddly specific.

**Comment by doug-101** [comment]
Yes, this is a floating point inaccuracy.  The actual result you get will vary by platforms (my Linux box gives zero).

There's not much I can do about it, since the app can't tell when the result should really be zero rather than a number very, very close to zero.

If you really don't want to see this, you could change the app settings to show results in "Fixed" notation, but that could be misleading when the results really should be extremely small numbers.
